cvat/apps/documentation/user_guide.md

```diff
@@ -171,27 +185,19 @@ Usage examples:
 Usage examples:
 - Edit attributes using keyboard with fast navigation between objects and frames.
 
-1. To enter into ``Attribute Annotation`` mode press ``Shift+Enter`` shortcut. After that it is possible to change attributes using keyboard.
+- In this Mode you can edit attributes with fast navigation between objects and frames using keyboard. Press ``Shift+Enter`` shortcut to enter AAMode. After that it is possible to change attributes using keyboard.
 
     ![](static/documentation/images/image023.jpg)
 
-2. The active attribute will be red. In this case it is ``Age``.
-
-    ![](static/documentation/images/image024.jpg)
-
-3. Look at the bottom side panel to see all possible shortcuts to change the attribute. Press ``4`` key on your keyboard to assign ``adult`` value for the attribute.
-
-    ![](static/documentation/images/image025.jpg)
+- The active attribute will be red. In this case it is ``Gender``. Look at the bottom side panel to see all possible shortcuts to change the attribute. Press ``2`` key on your keyboard to assign ``female`` value for the attribute.
 
-4. Press ``Up Arrow``/``Down Arrow`` keys on your keyboard to go to next attribute .
+    ![](static/documentation/images/image024.jpg) ![](static/documentation/images/image025.jpg)
 
-    ![](static/documentation/images/image026.jpg)
+- Press ``Up Arrow``/``Down Arrow`` keys on your keyboard to go to next attribute. In this case after pressing ``Down Arrow`` you will be able to edit ``Age`` attribute.
 
-5. In this case after pressing ``Down Arrow`` you will be able to edit ``Gender`` attribute.
+    ![](static/documentation/images/image026.jpg) ![](static/documentation/images/image027.jpg)
 
-    ![](static/documentation/images/image027.jpg)
-
-6. Use ``Right Arrow``/``Left Arrow`` keys to move on previous/next image.
+- Use ``Right Arrow``/``Left Arrow`` keys to move on previous/next image.
```

Comment:
Do we need mention, this keys do the search of frames which are satisfied for filter?  (default: non empty frames).

Reply:
It is specified in 'filter' section.